### PR TITLE
chore(apps/interface): update open graph image

### PR DESCRIPTION
**Pull request by math-marcellino**
Update open graph image (RIS-528)

image

### CODE REVIEW

File: apps/interface/pages/_app.tsx [hunk 1]
```diff
@@ -91,7 +91,7 @@ function App({ Component, pageProps }: AppProps) {
                     site_name: "Risedle",
                     images: [
                         {
-                            url: "https://risedle.com/assets/images/og/Landing.png",
+                            url: "/opengraph/LandingBSC.png",
```
Comment (risedledev) [inline apps/interface/pages/_app.tsx:94]: - Use full path for opengraph url
- Use template based string
- rename file to the following format `public/opengraph/$page-$chainslug.png`

For example:

Filenames:
-  `public/opengraph/landing-bsc.png`
- `public/opengraph/portfolio-bsc.png`
- `public/opengraph/trade-bsc.png`

then use it as below:

```typescript
url: `https://${chainSlug}.risedle.com/opengraph/landing-${chainSlug}.png",
```

you can get `chainSlug` from https://github.com/risedle/monorepo/blob/b4f3bd8a9f1cb44535076c4997179a9d788000a9/apps/interface/utils/getBaseConfig.ts#L41-L45

for example:

```
const { chainSlug } = getBaseConfig();
```

**Review by risedledev — CHANGES_REQUESTED** [review]

File: apps/interface/pages/_app.tsx (hunk 1 above)
Comment (pyk) [inline apps/interface/pages/_app.tsx:94]: - Use full path for opengraph url
- Use template based string
- rename file to the following format `public/opengraph/$page-$chainslug.png`

For example:

Filenames:
-  `public/opengraph/landing-bsc.png`
- `public/opengraph/portfolio-bsc.png`
- `public/opengraph/trade-bsc.png`

then use it as below:

```typescript
url: `https://${chainSlug}.risedle.com/opengraph/landing-${chainSlug}.png",
```

you can get `chainSlug` from https://github.com/risedle/monorepo/blob/b4f3bd8a9f1cb44535076c4997179a9d788000a9/apps/interface/utils/getBaseConfig.ts#L41-L45

for example:

```
const { chainSlug } = getBaseConfig();
```

**Review by pyk — CHANGES_REQUESTED** [review]

File: apps/interface/pages/_app.tsx (hunk 1 above)
Comment (math-marcellino) [reply apps/interface/pages/_app.tsx:94]: Got it sir, I'll update it real soon 👍

Comment (pyk) [reply apps/interface/pages/_app.tsx:94]: @math-marcellino I think we need to use term `home-bsc.png` instead of `landing-bsc.png`

Comment (math-marcellino) [reply apps/interface/pages/_app.tsx:94]: Okay sir, I'll rename it to 👍

Comment (math-marcellino) [reply apps/interface/pages/_app.tsx:94]: Btw should we add the `arbitrum` og image to this repo too? @pyk

Comment (pyk) [reply apps/interface/pages/_app.tsx:94]: @math-marcellino yess lets go.

Btw we need to wait from ayik first about this

Comment (pyk) [reply apps/interface/pages/_app.tsx]: @math-marcellino ok ok, all looks good. Just need to wait for the image

Comment (math-marcellino) [reply apps/interface/pages/_app.tsx]: The opengraph images are updated sir 👍

**Review by pyk — APPROVED** [review]

**Pull request merged**